tests/test_base.py

```diff
@@ -306,3 +306,10 @@ def test_math():
     assert -arcsech(sech(x)) == approx(x)
     assert arccsch(csch(x)) == approx(x)
     assert arccoth(coth(x)) == approx(x)
+
+def test_tensors():
+    """Test tensor variable names in calc.py"""
+    assert evaluator("U^{ijk}", {"U^{ijk}":2}, {}, {})[0] == 2
+    assert evaluator("U_{ijk}/2", {"U_{ijk}":2}, {}, {})[0] == 1
+    assert evaluator("U_{ijk}^{123}", {"U_{ijk}^{123}":2}, {}, {})[0] == 2
+    assert evaluator("U_{ijk}^2", {"U_{ijk}":2}, {}, {})[0] == 4
```

Comment:
Maybe add a test to show that powers still work?

```python
 assert evaluator("U_{ijk}^{123}^2", {"U_{ijk}^{123}":2}, {}, {})[0] == 4
```

Reply:
The fourth line already does this!

Reply:
OK, I wasn't sure if you wanted one that used both upper indices and powers. (4th doesn't use upper indices now).